src/script/interpreter.cpp

```diff
@@ -4,6 +4,7 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
+#include "main.h"
```

Comment:
We shouldn't be adding in dependency to main.h to this level of the code, it would be more appropriate to pass the proper flags through the arguments in the function.

For example, the check should be done outside of this function and only pass in `SCRIPT_VERIFY_CHECKLOCKTIMEVERIFY` if supermajority is active.